README.md

```diff
@@ -19,33 +19,26 @@ If you're using `brew`:
 $ brew install carina
 ```
 
-Otherwise you can grab a download from the links in [releases](https://github.com/getcarina/carina/releases), move it to somewhere on your `$PATH`, and make it executable.
+### Linux and OS X (non-homebrew)
 
-```bash
-$ mv carina ~/bin/carina
-$ chmod u+x ~/bin/carina
-```
-
-### Linux
-
-There are links to built binaries over in [releases](https://github.com/getcarina/carina/releases).
-
-After downloading the version for your system, you'll probably need to rename it,
-set it as executable, and put it on a `PATH` you have:
+Download a copy of `carina` straight from latest or via the links in [releases](https://github.com/getcarina/carina/releases), move it somewhere on your `$PATH`, and make it executable:
```

Comment:
I need someway to convey that we'll use latest in the following command. Wonder what I should change this to.